_layouts/home.html

```diff
@@ -40,7 +40,7 @@ <h2>The Java Multi-Cloud Toolkit</h2>
             </div>
 
             <div id="keystonev3" class="alert alert-success">
-                <p>If you are an OpenStack user, read our <a href="/blog/2018/01/16/keystone-v3/">last blog post</a> about the <strong>OpenStack Keystone V3</strong> integration that will be released in jclouds <strong>2.1.0</strong>!</p>
+                <p>If you are an OpenStack user, read our blog posts about the <a href="/blog/2018/01/16/keystone-v3/">OpenStack Keystone V3</a> and <a href="/blog/2018/02/06/nova-neutron">OpenStack Nova and Neutron</a> integrations that will be released in jclouds <strong>2.1.0</strong>!</p>
```

Comment:
Perhaps just "Read our blog posts about the..."? From the titles, it's pretty clear they're about OpenStack, so I don't think there's a risk of people clicking on them if the content is not relevant?